nextflow.config

```diff
@@ -247,7 +247,7 @@ manifest {
     description     = """A Nanostring nCounter analysis pipeline"""
     mainScript      = 'main.nf'
     nextflowVersion = '!>=23.04.0'
-    version         = '1.2.1dev'
+    version         = '1.2.1'
```

Comment:
Still dev I guess?

```suggestion
    version         = '1.2.1dev'
```

Reply:
No - will do immediate release after this